.babelrc

```diff
@@ -1,27 +1,10 @@
 {
   "plugins": [
     "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-transform-runtime",
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-transform-template-literals",
-    "@babel/plugin-proposal-object-rest-spread",
-    "@babel/plugin-transform-destructuring"
+    "@babel/plugin-transform-runtime"
```

Comment:
Why do we need `plugin-transform-runtime` and `plugin-syntax-dynamic-import`?

Reply:
All the other plugins we had there are already included at the `@babel/preset-env`, those two aren't. But let me give some details:

About `plugin-transform-runtime`, this one, according to [Babel docs](https://babeljs.io/docs/en/babel-plugin-transform-runtime), avoid helpers duplication and I think it's a good idea to keep it.

About `plugin-syntax-dynamic-import`, this is the one that allows us to use `import` syntax and though [Babel docs](https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import) says it's included at `preset-env`, I ran a small script to check which plugins were included and this one wasn't.